src/sections/General/Footer/footer.style.js

```diff
@@ -232,6 +232,33 @@ const FooterWrapper = styled.section`
 			padding: 0 1rem;
 		}
 	}
+	
+	
+	
+	@media screen and (max-width: 445px) {
+           .terms{
+              position: relative;
+              top: -32px;
+              left: 30px;
+            }
+	     .privacy{
+               position: relative;
+               left: -28px;
+             }
+	     .status{
+              position: relative;
+              left: -28px;
+            }
+	     
+           }
```

Comment:
There is still an overlap in mobile view.
So, btter solution is reduce the width/height of the `scroll-to-top` button in mobile view and increase the bottom value such that the button lies in between the subscribe button the policies. And, increase the top margin a bit for the terms, status line.